partiql-planner/src/test/kotlin/org/partiql/planner/internal/typer/PlanTyperTestsPorted.kt

```diff
@@ -760,22 +760,19 @@ class PlanTyperTestsPorted {
             ErrorTestCase(
                 name = "BITWISE_AND_MISSING_OPERAND",
                 query = "1 & MISSING",
-                expected = unionOf(INT4, INT8, INT),
+                expected = ANY, // TODO: Is this unionOf(INT4, INT8, INT) ?
```

Comment:
Hmm thought it should also be the `union(INT4, INT8, INT)` #1463. May need to check this again?

Reply:
Agreed -- however, I'd like to just get this in. Realistically, if we want to narrow down the type, this can be accomplished using the "unknown" type that PostgreSQL exposes.

Reply:
Could consider capturing the future addition of an "unknown" type in a GH issue. Since this is just a merge, we can punt on the behavior on this (and similar tests).